### PR TITLE
Move breadcrumbs

### PR DESCRIPTION
**Pull request by kosiakkatrina**
Move the breadcrumbs under the phase banner

image

### CODE REVIEW

File: app/views/case_logs/edit.html.erb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <% content_for :title, "Log #{@case_log.id}" %>
+<% breadcrumbs = { "Logs" => "/logs", content_for(:title) => "" } %>
+<% content_for :breadcrumbs,  govuk_breadcrumbs(breadcrumbs: breadcrumbs)%>
```
Comment (paulrobertlloyd) [inline app/views/case_logs/edit.html.erb]:
```suggestion
<% content_for :breadcrumbs,  govuk_breadcrumbs(breadcrumbs: breadcrumbs) %>
```

File: app/views/form/check_answers.html.erb [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <% content_for :title, "#{subsection.id.humanize} - Check your answers" %>
+<% breadcrumbs = { "Logs" => "/logs", "Log" => "/logs/" + @case_log.id.to_s, subsection.id.humanize => "" } %>
+<% content_for :breadcrumbs, govuk_breadcrumbs(breadcrumbs: breadcrumbs)%>
```
Comment (paulrobertlloyd) [inline app/views/form/check_answers.html.erb]: With apologies for my whitespace OCD 🤪

```suggestion
<% content_for :breadcrumbs, govuk_breadcrumbs(breadcrumbs: breadcrumbs) %>
```

**Review by paulrobertlloyd — APPROVED** [review]

File: app/views/form/check_answers.html.erb (hunk 2 above)
Comment (kosiakkatrina) [reply app/views/form/check_answers.html.erb]: completely valid!

**Comment by paulrobertlloyd** [comment]
🙏

**Pull request merged**